### DISCUSSION TITLE
Welcome new Godot Community Games Org members!

### DISCUSSION THREAD

**Discussion by GodotCommunityGames** [Godot-Community-Games-Org/game1#1] [Announcements]
Stop by and say hi!

We can use this Discussions section initially to start to figure out how this will all work. And also decide if this Github Discussions is fine for communication or if we want to move to another communication platform.

I'm Todd by the way.

**Comment by SpearsDracona** [comment]
Hi, I'm Angel. I'm fairly new to working with Godot. I'm mainly an artist but I've been learning gdscript and figuring out how to make games with Godot. The idea of a game made by a bunch of people in the community sounds chaotic in a fun way and definitely sounds like something I want to be a part of. 

I'm thinking a platform like Discord might be more ideal for discussions but that might just be because I'm not used to GitHub. Anyways, nice to meet you and I can't wait to see what this becomes!

**Reply by GodotCommunityGames** [reply]
Hi Angel! I'm sure artists will be valuable (as a dev who has very little art skills!), and of course you can stretch your nascent programming skills here too. I used to be a software dev years ago but I'm only a month into learning Godot and game dev. I'm sure we'll get very experienced people to new people like us. Hopefully it'll be a good learning and community experience, and hopefully we'll actually finish some fun games!

Yeah totally Discord could work. Already had several people suggest that. I figure maybe we just get people added here first and then discuss setting up a discord. I've never really used Discord so I don't even know what you gotta do to set one up or in general how it works haha, potentially someone else who joins and actually uses Discord could set it up. But we can figure that out in the coming days as we start to get this community set up.

**Reply by StrawberrySmoothieDev** [reply]
> could

If you want I could set one up now, then transfer ownership to u. Idk if you already have a discord account but that's an option

**Reply by ryankmims** [reply]
> > could
> 
> If you want I could set one up now, then transfer ownership to u. Idk if you already have a discord account but that's an option

Can you set one up and then post it in the discussion board. Then maybe one of our first community updates to the main branch could be updating the readme with a link to the channel?

**Reply by StrawberrySmoothieDev** [reply]
> > > could
> 
> > 
> 
> > If you want I could set one up now, then transfer ownership to u. Idk if you already have a discord account but that's an option
> 
> 
> 
> Can you set one up and then post it in the discussion board. Then maybe one of our first community updates to the main branch could be updating the readme with a link to the channel?

Here you go. https://discord.gg/uQ3fe43hdc

**Reply by GodotCommunityGames** [reply]
Yup ryan I just posted the discord invite link on the readme here

**Comment by Yonodactyl** [comment]
Hey all! I'm Yon - iOS Developer with a passion for game dev. I'm a little bit of jack of all trades, master of none. I have started well over 5 games with Godot but have never finished a single one lol.

Looking forward to building some cool stuff and learning something new along the way. I'm realizing now that this is kind of like a little collective (like the Sokpop guys)!

**Comment by ryankmims** [comment]
I'm Ryan. I'm a software engineer and musician. Currently working on my first game in Godot and [have gotten pretty far in its development](https://www.youtube.com/watch?v=dPdEUngh9r0). Shooting for a pre-alpha by January 1, 2025.

My biggest strengths are programming and music. I'm really big on design as well, but where I'm lacking is art (if you watched the intro to that video I linked, you'll see what I mean).

So I would love to contribute where my strengths are, and hopefully learn some really cool stuff in the areas I'm lacking. I can crack open Blender if you really need me to, but you'll probably want me to open Ableton instead lol

**Comment by StrawberrySmoothieDev** [comment]
Heya! I'm smoothie, a long time programmer and godot dev with a bunch of experience in the nicher sides of godot. I'm currently working on a space exploration game called [Endless Abyss](https://smoothiegames.carrd.co) and also have a very large (horrendously coded) private 2D project called shocknawe. I've got a good bit of art experience primarily doing 3D models for my various projects, but my 2D are skills leave quite a bit to be desired. My specialties primarily focus on optimization, graphical pizazz, fancy backend systems, math, and algorithms, and wrangling godot's [scuffed compute shader implementation](https://app.milanote.com/1SJ1Nd16j7xv2T?p=0ULmZcMVtYC) :3 I defo struggle with direction and design, as well as planning things out, but I'm working on that by forcing myself to write stuff down. As for the communication method, I'd vote for a discord personally.

**Comment by StrawberrySmoothieDev** [comment]
DISCORD HERE: https://discord.gg/uQ3fe43hdc

**Comment by R1cHero** [comment]
Hi! I'm Emre, I have been developing 2D games in Godot for 8 months. I love doing research and learning new things. It seems like fun to meet new people and develop games with many people from the community.

**Comment by coderKillo** [comment]
Hi! I'm killo, I have been a software developer for over 5 years now and started game development as a hobby about 2 years ago. Since then I have learned alot in art, design, sound, etc. This year I participated in my first game jam (GMTK Game Jam) and my first (finished) game [Melt Them All](https://store.steampowered.com/app/3252500/Melt_Them_All/) is releasing on steam soon. Now I need something new and because in the passt 2 years I developed games solo, I really want to know how it is to create a game in a team.

**Comment by DaRaKu2002** [comment]
Henlo! Im David. 
Been learning godot for over half a year now. Started on a 2D top down shooter, participated in a jam once and am now working on a 3D plattformer. I am also a coding novice and dont have any 2D Art skills and minimal Blender knowledge.
I would really like to participate in a 'bigger then myself' project and contribute in anyway I can.

**Comment by bubba169** [comment]
Hi I'm Simon. I've been working in games and web dev for a fair few years since I finished my games development course at Uni. I have commercial experience on many projects and areas of development including C++, Unity (C#), AS3/Haxe, JS and full stack PHP/MySQL. 

These days, I'm mainly building 2D web games with Haxe/JS/PHP, but I recently found Godot and wanted to try building something with it as a hobby. I've not got much experience with Godot but I'm hoping my existing dev knowledge is transferrable. I've also been playing about with Blender and learning 3D modelling so that could come in handy.

**Comment by ghost** [comment]
Hello everyone I'm Daniel,
I'm mostly a 2D artist/animator but I have a new found interest in coding (though I'm still very new to it!), I have way too many solo projects than is healthy but I really want to collaborate because it's more fun :) 

Also I need to get used to this GitHub sorcery and I figured this would be a good way to practice it. Seems like we have quite few guys here that are familiar with the software development cycle and I'm going to absorb their skills through osmosis !
